### DISCUSSION TITLE
Show the controls of Installed on the details page of Browse, when installed?

### DISCUSSION THREAD

**Discussion by HubKing** [mjakeman/extension-manager#343] [Ideas]
Currently, when you install an extension by clicking "Install" in the search result, there is nothing you can do. You have to click the back arrow (why doesn't mouse Backward button work for this?), click the "Installed" tab, and find the extension that you have just installed and click the settings/uninstall button  there. This is very inconvenient and tedious.

Why not show the controls (enable/disable, settings, uninstall, etc) on the same page so that one could easily install an extension, test it, and uninstall it if it is not the thing one was looking for?

![image](https://user-images.githubusercontent.com/10900489/218286793-d1e4e1a8-568c-405d-94cd-6bf259a2c15e.png)

**Comment by mjakeman** [comment]
> Why not show the controls (enable/disable, settings, uninstall, etc) on the same page so that one could easily install an extension, test it, and uninstall it if it is not the thing one was looking for?

Tracked as #197 - I'm intending to have this out for next release (v1.0)

> why doesn't mouse Backward button work for this?

I don't have a mouse with a backwards button, so I can't test it. We use libadwaita's built in Leaflet widget, so perhaps you might want to suggest it on that repo? https://gitlab.gnome.org/GNOME/libadwaita

**Reply by oscfdezdz** [reply]
> why doesn't mouse Backward button work for this?

It has been working for a while now, since we switched to `AdwNavigationPage`.
